lib/puppet/application/filebucket.rb

```diff
@@ -148,6 +158,17 @@ def restore
     @client.restore(file, md5)
   end
 
+  def diff
+    cksm_a = args.shift
+    cksm_b = args.shift || ''
+    if cksm_a
```

Comment:
Should this be `cksm_b`?

Can we have longer variable names? :)

Also, is the empty string a good default?